go.mod

```diff
@@ -13,17 +13,17 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-logr/logr v0.1.0
 	github.com/go-logr/zapr v0.1.1
-	github.com/google/go-cmp v0.3.1
+	github.com/google/go-cmp v0.4.0
 	github.com/kevinburke/go-bindata v3.11.0+incompatible
+	github.com/onsi/gomega v1.10.1
```

Comment:
I see this PR adds the `gomega` package for `pkg/util/network_test.go`. This adds ~4Mb to the `vendor/` folder.
Is there a compelling reason to pull in `gomega`? I'm not opposed to having it, but am just wondering if there's any plans to use it more within unit tests, or something along those lines.

Reply:
@sgreene570 this was on my mind, so thanks for bringing it up. @Miciah thoughts ^

Reply:
@sgreene570 commit `3f1bfad` removes `gomega` in favor of table tests.